.pylintrc

```diff
@@ -33,7 +33,7 @@ unsafe-load-any-extension=no
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code
-extension-pkg-whitelist=libtorrent,netifaces
```

Comment:
Thanks for the review! 💪 

I believe this is leftover code from Dispersy from roughly 15 years ago 😄 `libtorrent` is not referenced/used in IPv8. This can be safely removed.

Short story long: initially, I touched this line because I removed `netifaces` entirely. Then, I undid my change partially (adding `netifaces` again for backward compatibility) but this small cleanup stuck around.